### PR TITLE
add new kleinanzeigen logo

### PR DESCRIPTION
**Pull request by peterge1998**
## Description

Fixes #(issue)

## Type of change

:x: Bug fix (non-breaking change which fixes an issue)
:x: Icon addition (non-breaking change that adds/modifies Lawnicons's icons)
:x: General change (non-breaking change that doesn't fit the above categories like copyediting)

### Icons added
* App Name (`package.name`) - added from icon request form
* App Name (`package.name`)
* App Name (`package.name`)

### Icons updated
* App Name (`package.name`)
* App Name (`package.name`)

### Icons linked
* App Name (linked `package.name` to `@drawable/package`)
* App Name (linked `package.name` to `@drawable/package`)

### CODE REVIEW

File: svgs/bob.svg
Comment (Chefski) [inline svgs/bob.svg:1]: image

The width for the B's is too big. 14px is the maximum

File: svgs/rewe.svg
Comment (Chefski) [inline svgs/rewe.svg:1]: The icon is slightly bigger than 148px. Mind making it smaller?

**Review by Chefski — CHANGES_REQUESTED** [review]

File: svgs/rewe.svg
Comment (peterge1998) [reply svgs/rewe.svg:1]: Done

File: svgs/bob.svg
Comment (peterge1998) [reply svgs/bob.svg:1]: Done

**Review by Chefski — APPROVED** [review]

**Pull request merged**